contributing/tech_enhancement.rst

```diff
@@ -1,6 +1,9 @@
 Suggest a Technical Enhancement
 ===============================
 
-* Fork https://github.com/akeneo/pim-community-dev
-* Describe the improvement, write some sample code
-* Submit a Pull Request that allows to discuss the enhancement
+* Check if this enhancement is already discussed: https://github.com/akeneo/pim-community-dev/labels/tech-enhancement
+* Create a new github issue that:
+
+  * clearly describes the enhancement in its title
+  * Describe the improvement, write some sample code
+  * Submit a Pull Request that allows to discuss the enhancement
```

Comment:
You should extract this from the list and make it a sentence otherwise it doesn't work and you'd have "Create a new GitHub issue that: submit a pull request"